### PR TITLE
Update UG with edit and fix wrong usage message

### PR DESCRIPTION
**Pull request by yiwen101**
Regarding the bug:
Old message
Screenshot 2023-11-11 at 20 23 31

New message:
Screenshot 2023-11-11 at 20 23 58

### CODE REVIEW

File: docs/TempUG.md [hunk 1]
```diff
@@ -545,93 +546,118 @@ Please use show lessonIndex before deleting task!
...
-</box>
-
+Edits the specified item in the student/schedule list of the application.
 
-#### For Student:
-<!-- use -subject SUBJECT for student -->
-Format: `command COMPULSORY [optional]` (for list specific format)
-* Format info 1
-* Format info 2
+#### For student list:
```
Comment (pzl111) [inline docs/TempUG.md]: i think need to change to #### For Student:

**Review by pzl111 — COMMENTED** [review]
minor change

File: docs/TempUG.md (hunk 1 above)
Comment (yiwen101) [reply docs/TempUG.md]: Required update made. Thank you for pointing that out

File: docs/TempUG.md [hunk 2]
```diff
@@ -545,93 +546,118 @@ Please use show lessonIndex before deleting task!
...
-
-<box type="tip" seamless> 
-
-**Tips:**
-- Tip 1
-- Tip 2
-
-</box>
-
+Edits the specified item in the student/schedule list of the application.
```
Comment (richiehx) [inline docs/TempUG.md]: Maybe should change to the ___STUDENTS list___ and ___SCHEDULE list___?

File: docs/TempUG.md [hunk 3]
```diff
@@ -545,93 +546,118 @@ Please use show lessonIndex before deleting task!
...
 Failure outputs:
-* Input: `invalid command code here`
-  * Error: Explanation and solution here, this is because the flag has an incorrect value, bla bla bla
+* Input: `editLesson 1`
+  * Error: No field to edit given. Need to enter at least one field to edit.
 ```
-Invalid command with the error message here
+No edit detected. Please edit at least one field: name, start, end, subject, day to different value.
+```
+Failure outputs:
```
Comment (richiehx) [inline docs/TempUG.md]: I think you don't need to write the Failure outputs heading again.

File: docs/TempUG.md [hunk 4]
```diff
@@ -545,93 +546,118 @@ Please use show lessonIndex before deleting task!
...
+* Input: `editLesson -name a new name` (assuming no lesson is currently shown)
+  * Error: No lesson shown. Show a lesson with the `show` command.
+```
+Using edit command without specifying index when no entry is shown. 
+Usage: edit [INDEX] (at least one of unique [-name|subject|day|start|end VALUE]). 
+For example, edit 1 -name lesson2 -subject English -day 23/12 -start 14:30 -end 16:30
+If you want to edit the currently shown lesson, you could omit the index. 
+Note your edited 'name' must not already in the schedule and 'start' must be before 'end'.
+```
+Failure outputs:
```
Comment (richiehx) [inline docs/TempUG.md]: Same as above.

**Review by richiehx — COMMENTED** [review]
LGTM, just some nits for the UG docs.

File: docs/TempUG.md (hunk 2 above)
Comment (yiwen101) [reply docs/TempUG.md]: Thank you for pointing that out. Required update made.

File: docs/TempUG.md (hunk 3 above)
Comment (yiwen101) [reply docs/TempUG.md]: Thank you for pointing that out. Required update made.

File: docs/TempUG.md (hunk 4 above)
Comment (yiwen101) [reply docs/TempUG.md]: Thank you for pointing that out. Required update made.

**Review by richiehx — APPROVED** [review]
LGTM!

**Pull request merged**